datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxDatabaseBackupStatusResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.sequenceiq.sdx.api.model;
+
+public class SdxDatabaseBackupStatusResponse {
+    private String status;
```

Comment:
what is the value set? If it is fixed then I would prefer using an enum, if not then it is fine.

Reply:
backupId is the UUID for the backup. It is the Id generated by datalake-dr service. Yes, the name is the stack name. There is no use for backup name here.